<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/res.qrc">
    <normaloff>:/icons/app</normaloff>:/icons/app</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableView" name="tableViewConfig">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Con&amp;nection</string>
    </property>
    <addaction name="actionDisconnect"/>
    <addaction name="actionTestLatency"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>&amp;Config</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>&amp;Add</string>
     </property>
     <property name="icon">
      <iconset resource="../../res/res.qrc">
       <normaloff>:/icons/actions/list-add</normaloff>:/icons/actions/list-add</iconset>
     </property>
     <addaction name="actionAddManually"/>
     <addaction name="actionAddFromURI"/>
    </widget>
    <addaction name="menuAdd"/>
    <addaction name="actionConnect"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
    <addaction name="actionShare"/>
    <addaction name="actionRefresh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfig"/>
   <addaction name="menuConnection"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QDockWidget" name="dockWidgetConnections">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Connections</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTableView" name="tableViewConnections">
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowserLog">
       <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarConfig">
   <property name="windowTitle">
    <string>Config Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionEdit"/>
   <addaction name="actionRemove"/>
   <addaction name="actionShare"/>
   <addaction name="actionRefresh"/>
  </widget>
  <widget class="QToolBar" name="toolBarConnection">
   <property name="windowTitle">
    <string>Connection Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDisconnect"/>
   <addaction name="actionTestLatency"/>
  </widget>
  <widget class="QToolBar" name="toolBarFile">
   <property name="windowTitle">
    <string>File Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/network-connect</normaloff>:/icons/actions/network-connect</iconset>
   </property>
   <property name="text">
    <string>&amp;Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/network-disconnect</normaloff>:/icons/actions/network-disconnect</iconset>
   </property>
   <property name="text">
    <string>&amp;Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/document-edit</normaloff>:/icons/actions/document-edit</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
   <property name="toolTip">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/document-share</normaloff>:/icons/actions/document-share</iconset>
   </property>
   <property name="text">
    <string>&amp;Share</string>
   </property>
   <property name="toolTip">
    <string>Share</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/document-import</normaloff>:/icons/actions/document-import</iconset>
   </property>
   <property name="text">
    <string>&amp;Import From gui-config.json</string>
   </property>
   <property name="toolTip">
    <string>Import From gui-config.json</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/document-export</normaloff>:/icons/actions/document-export</iconset>
   </property>
   <property name="text">
    <string>&amp;Export To gui-config.json</string>
   </property>
   <property name="toolTip">
    <string>Export To gui-config.json</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/application-exit</normaloff>:/icons/actions/application-exit</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/help-about</normaloff>:/icons/actions/help-about</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/qt</normaloff>:/icons/actions/qt</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAddManually">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/document-new</normaloff>:/icons/actions/document-new</iconset>
   </property>
   <property name="text">
    <string>Add Config &amp;Manually</string>
   </property>
   <property name="toolTip">
    <string>Add Config Manually</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/list-remove</normaloff>:/icons/actions/list-remove</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/view-refresh</normaloff>:/icons/actions/view-refresh</iconset>
   </property>
   <property name="text">
    <string>Re&amp;fresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionAddFromURI">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/insert-link</normaloff>:/icons/actions/insert-link</iconset>
   </property>
   <property name="text">
    <string>Add Config From &amp;URI</string>
   </property>
   <property name="toolTip">
    <string>Add Config From URI</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/window</normaloff>:/icons/actions/window</iconset>
   </property>
   <property name="text">
    <string>&amp;Show</string>
   </property>
   <property name="toolTip">
    <string>Show Window</string>
   </property>
  </action>
  <action name="actionTestLatency">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/chronometer</normaloff>:/icons/actions/chronometer</iconset>
   </property>
   <property name="text">
    <string>&amp;Test Latency</string>
   </property>
   <property name="toolTip">
    <string>Test Latency</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../res/res.qrc">
     <normaloff>:/icons/actions/configure</normaloff>:/icons/actions/configure</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
